.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,67 @@
+name: Bug Report
+description: Report a bug impacting AVD collection
+title: "Bug Report Title"
+labels: ['type: bug', triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: textarea
+    id: issue-summary
+    attributes:
+      label: Issue Summary
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "please detail your issue"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: component
+    attributes:
+      label: Which component of AVD is impacted
+      multiple: false
```

Comment:
issues with plugins could impact multiple components.

Reply:
We can activate `multiple: true` or also list plugins.